.github/workflows/pr.yaml

```
@@ -1,17 +1,38 @@
 name: PR Test operator
 
 on:
-  pull_request:
-    branches: 
-    - main
-    - rhdh-1.[0-9]+
-    - 1.[0-9]+.x
-    - release-1.[0-9]+
+  pull_request_target:
+      types: [opened, synchronize, reopened, ready_for_review]
+      branches: 
+      - main
+      # - rhdh-1.[0-9]+
+      # - 1.[0-9]+.x
+      # - release-1.[0-9]+
```

```suggestion
      - rhdh-1.[0-9]+
      - 1.[0-9]+.x
      - release-1.[0-9]+
```

Should be uncommented.